<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-3V017FK" failures="0" tests="4" name="listener.SampleTest" time="0.537" errors="1" timestamp="2023-06-29T16:45:45 IST" skipped="1">
  <testcase classname="listener.SampleTest" name="TC1" time="0.002">
    <system-out>
      <![CDATA[test TC1execution start
TC1 is Rinning
thisTC1 method is compleated successfully]]>
    </system-out>
  </testcase> <!-- TC1 -->
  <testcase classname="listener.SampleTest" name="TC3" time="0.000">
    <skipped/>
    <system-out>
      <![CDATA[thisTC3 Tc is skipped.. please check dependent method ]]>
    </system-out>
  </testcase> <!-- TC3 -->
  <testcase classname="listener.SampleTest" name="TC4" time="0.001">
    <system-out>
      <![CDATA[test TC4execution start
TC4 is Rinning
thisTC4 method is compleated successfully]]>
    </system-out>
  </testcase> <!-- TC4 -->
  <testcase classname="listener.SampleTest" name="TC2" time="0.534">
    <error message="Method listener.SampleTest.TC2() didn&#039;t finish within the time-out 500" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method listener.SampleTest.TC2() didn't finish within the time-out 500
at java.base@18.0.2.1/java.lang.Thread.sleep(Native Method)
at app//listener.SampleTest.TC2(SampleTest.java:18)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$DMH/0x0000000800d19c00.invokeVirtual(LambdaForm$DMH)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$MH/0x0000000800d1a400.invoke(LambdaForm$MH)
at java.base@18.0.2.1/java.lang.invoke.Invokers$Holder.invokeExact_MT(Invokers$Holder)
at java.base@18.0.2.1/jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(DirectMethodHandleAccessor.java:154)
at java.base@18.0.2.1/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base@18.0.2.1/java.lang.reflect.Method.invoke(Method.java:577)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@18.0.2.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
    <system-out>
      <![CDATA[test TC2execution start
thisTC2 method is failed please try again]]>
    </system-out>
  </testcase> <!-- TC2 -->
  <system-out>
    <![CDATA[test TC2execution start
thisTC2 method is failed please try again
test TC1execution start
TC1 is Rinning
thisTC1 method is compleated successfully
test TC4execution start
TC4 is Rinning
thisTC4 method is compleated successfully
test TC3execution start
thisTC3 Tc is skipped.. please check dependent method ]]>
  </system-out>
</testsuite> <!-- listener.SampleTest -->
